packages/now-cli/src/commands/domains/index.ts

```diff
@@ -85,6 +92,7 @@ export default async function main(ctx: NowContext) {
       '--no-cdn': Boolean,
       '--yes': Boolean,
       '--next': Number,
+      '-N': '--next',
```

Comment:
Let's remove `-N` like I mentioned in the other PR.

Reply:
@styfle Please check https://github.com/zeit/now/pull/3915#discussion_r395814474